$MirOS: ports/audio/libvorbis/patches/patch-Makefile_am,v 1.1 2005/11/09 19:44:59 bsiegert Exp $
--- Makefile.am.orig	Mon Nov 28 05:43:03 2005
+++ Makefile.am	Wed Dec 28 16:00:30 2005
@@ -2,7 +2,7 @@
 
 AUTOMAKE_OPTIONS = foreign dist-zip
 
-SUBDIRS = lib include doc examples vq
+SUBDIRS = lib include doc #examples vq
 
 m4datadir = $(datadir)/aclocal
 m4data_DATA = vorbis.m4
